test api/REST Client/API endpoints.http

```diff
@@ -0,0 +1,139 @@
+@environmentName = Sandbox1
+@aPIPublisher = ciellos
+@aPIGroup = salesOrderApi
+@aPIVersion = v1.0
+@baseAPIUrl = {{baseUrl}}/v2.0/{{environmentName}}/api/{{aPIPublisher}}/{{aPIGroup}}/{{aPIVersion}}
+@entitySetName = items
+
+###########################################################################
+#    ____      _                                   _        _              
+#   / ___| ___| |_    __ _  ___ ___ ___  ___ ___  | |_ ___ | | _____ _ __  
+#  | |  _ / _ \ __|  / _` |/ __/ __/ _ \/ __/ __| | __/ _ \| |/ / _ \ '_ \ 
+#  | |_| |  __/ |_  | (_| | (_| (_|  __/\__ \__ \ | || (_) |   <  __/ | | |
+#   \____|\___|\__|  \__,_|\___\___\___||___/___/  \__\___/|_|\_\___|_| |_|
+#
+###########################################################################
+# @name tokenrequest
+POST https://login.microsoftonline.com/{{tenant}}/oauth2/v2.0/token
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=client_credentials
+&client_id={{clientid}}
+&client_secret={{clientsecret}}
+&scope={{scope}}
+###
+@token = {{tokenrequest.response.body.access_token}}
+###
+@baseUrl = https://api.businesscentral.dynamics.com
+##############################################################################
+### GET all APIs
+# @name allApi
+GET {{baseUrl}}/v2.0/{{environmentName}}/api/v2.0/
+Authorization: Bearer {{token}}
+#
+
+### GET $metadata
+# @name $metadata
+GET {{baseUrl}}/v2.0/{{environmentName}}/api/v2.0/$metadata
+Authorization: Bearer {{token}}
+#
+
+### GET companies
+# @name companies
+GET {{baseUrl}}/v2.0/{{environmentName}}/api/v2.0/companies
+Authorization: Bearer {{token}}
+#
+
+### GET items
+### Instead of using "{{companies.response.body.value[0].id}}", it's possible to use companyId like "3ac0aadc-5ae6-ee11-a201-6045bde98aeb"
+# @name items
+GET {{baseUrl}}/v2.0/{{environmentName}}/api/v2.0/companies({{companies.response.body.value[0].id}})/{{entitySetName}}
+Authorization: Bearer {{token}}
+#
+
+### GET items with select, top and skip
+# @name itemsSelect
+GET {{baseUrl}}/v2.0/{{environmentName}}/api/v2.0/companies({{companies.response.body.value[0].id}})/{{entitySetName}}?$select=number,displayName,unitCost&$top=5&$skip=10
+Authorization: Bearer {{token}}
+#
+
+### GET items using filters
+# @name itemsFilter
+GET {{baseUrl}}/v2.0/{{environmentName}}/api/v2.0/companies({{companies.response.body.value[0].id}})/{{entitySetName}}?$filter=type eq 'Inventory' and unitPrice lt 125.1
+Authorization: Bearer {{token}}
+#
+
+### GET items using orderby
+# also possible to use desc
+# @name itemsSort
+GET {{baseUrl}}/v2.0/{{environmentName}}/api/v2.0/companies({{companies.response.body.value[0].id}})/{{entitySetName}}?$orderby=displayName asc
+Authorization: Bearer {{token}}
+#
+
+### GET items using pagination
+# @name itemsPagination
+GET {{baseUrl}}/v2.0/{{environmentName}}/api/v2.0/companies({{companies.response.body.value[0].id}})/{{entitySetName}}
+Authorization: Bearer {{token}}
+Prefer: odata.maxpagesize=2
```

Comment:
You can then use that variable @pageSize here

Reply:
Code reviewed.